server/container_create.go

```diff
@@ -976,21 +1039,39 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 			return nil, err
 		}
 
+		mnt = rspec.Mount{
+			Type:        "bind",
+			Source:      sb.ResolvPath(),
+			Destination: "/etc/resolv.conf",
+			Options:     options,
```

Comment:
every call of `AddBindMount` that you've replaced with `AddMount`, you now need to add either `bind` (or `rbind`) to `options` when it's a bind mount (here for this case for instance https://github.com/kubernetes-incubator/cri-o/pull/1244/files#diff-dba9598e19f7903533dc9add98d410d2R1033). Cheers :)

Reply:
You can take a look at how it was `AddBindMount` here https://github.com/opencontainers/runtime-tools/pull/279/files#diff-ed9e41f69ceb4f9d141e2fe7a9da68a4L914

Reply:
@rhatdan ^^